incubator/group/keeper_test.go

```diff
@@ -19,14 +22,15 @@ import (
 func TestCreateGroup(t *testing.T) {
 	amino := codec.New()
```

Comment:
I would move all these declarations into a reusable `createTest()`

Reply:
Thanks for the suggestion. My personal preference is that I try to keep the tests self explaining without setup functions mostly. A bit setup in a tests may also tell a story that there is something wrong with the architecture or test architecture.